resolwe/permissions/filters.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from rest_framework.filters import BaseFilterBackend
+
+from resolwe.permissions.shortcuts import get_objects_for_user
+
+
+class ResolwePermissionsFilter(BaseFilterBackend):
+
+    def filter_queryset(self, request, queryset, view):
+        user = request.user
+
+        app_label = queryset.model._meta.app_label
+        model_name = queryset.model._meta.model_name
+
+        kwargs = {}
+        if model_name == 'storage':
```

Comment:
Could this be changed so that it (`model_name` and `perms_filter`) is configured on the viewset instead of being hardcoded here?